.github/workflows/sanity-test.yaml

```diff
@@ -103,14 +103,17 @@ jobs:
         run: |
           ./corso backup create exchange \
           --user "${CORSO_M365_TEST_USER_ID}" \
-           --hide-progress 2>&1 | tee $TEST_RESULT/backup_exchange.txt 
+           --hide-progress --json 2>&1 | tee $TEST_RESULT/backup_exchange.txt 
 
-          if ! grep -q 'Completed' $TEST_RESULT/backup_exchange.txt
-          then
+          resultjson=$(sed -e '1,/Completed Backups/d' $TEST_RESULT/backup_exchange.txt ) 
```

Comment:
We can remove this in a follow up once we change the CLI not to output anything if --json or --json debug is specified - it breaks the ability to just pipe the output to jq